{"uuid": "e7ac2f23-43dd-4a40-8180-59fb0f780814", "children": ["12fa3621-51cf-4fae-b48c-f8c86b11a88f", "30e18ad7-4316-4857-8f63-e2d9f3f5b13f"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'ifname'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 403, in get_interface_details\n    wifi_info[j[\"ifname\"]] = [j[\"config\"][\"ssid\"], j[\"config\"][\"encryption\"], \"\"]\n"}, "attachments": [{"name": "ucentral_config: ", "source": "39a85b0b-0d14-4572-bf22-63275abb815f-attachment.attach"}, {"name": "Time Took to apply Config: 56.871309995651245", "source": "9912aee3-fa0c-4b1e-8280-a72391dde40c-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "f6169e9c-9577-4e3d-8526-ea7047a7ed3c-attachment.attach"}, {"name": "AP LOgs: ", "source": "4566fd69-6489-45d8-8929-959581899ad8-attachment.attach"}], "start": 1629160421297, "stop": 1629160516213}], "start": 1629160421297, "stop": 1629160517023}